flink-core/src/main/java/org/apache/flink/util/function/FunctionUtils.java

```diff
@@ -115,4 +116,14 @@ private FunctionUtils() {
 			return result;
 		};
 	}
+
+	/**
+	 * Converts a {@link RunnableWithException} into a {@link Callable} that will return the {@code result}.
+	 */
+	public static <T> Callable<T> asCallable(RunnableWithException command, T result) {
```

Comment:
Could we move that static function to a default member function of RunnableWithException?

Reply:
`RunnableWithException` is `@Public`, while `FunctionUtils` are not. That's why I placed it here.

Reply:
Even more reason to add something that is useful for the all users. ;)

Reply:
ok, I will move it to the `RunnableWithException`.